### DISCUSSION TITLE
Leveling Tracker Custom Notes

### DISCUSSION THREAD

**Discussion by baburu** [Lailloken/Exile-UI#245] [Ideas & Feedback]
Hey there, I wanted to use this amazing Leveling Tracker you made, and I was hoping to use my custom notes; but yesterday while messing with it, I realized that I can't manipulate the behaviour of the Tabs that get automatically created once you Import the guide.

For example I wanted to have 5 lines per Tab and manually switch them, and I even tried to manipulate the behaviour by using Arrows formatting from Exile Leveling, but the Tracker just changes the number of lines displayed automatically.

Would it be possible to have a Manual mode for people who want to have just basically Notes and being able to switch tabs manually?

Thanks in advance.

**Comment by Lailloken** [comment]
The tracker is specifically built around automation, and taking that out would require fundamental changes. HeartofPhos told me that they don't expect many people to use edited guides/routes, so I won't put any more work than necessary into it if it's just for a tiny sub-group of users. The script parses all the steps in a guide/route and groups them together in a way that the final bullet-point in a panel contains a location. If the player enters that location, the script knows that it has to switch to the next panel.

Depending on how many notes you're intending to use, you could use my notepad feature instead (though you would have to manually type your notes, as exile-leveling doesn't support copy-pasting):

![image](https://user-images.githubusercontent.com/61888437/233069453-4d1f8794-7142-4920-bf28-35033548ee93.png)
![image](https://user-images.githubusercontent.com/61888437/233069557-c31c1d44-6e1a-4e41-b9b6-3dfd879c8d89.png)
![image](https://user-images.githubusercontent.com/61888437/233069639-b6f5d104-b55a-4beb-9854-50020ccba609.png)

**Reply by baburu** [reply]
I see thanks anyways for the reply and for your work.
